conf/summariser.cfg

```diff
@@ -6,3 +6,20 @@ pidfile = /var/run/apel/summariser.pid
 logfile = /var/log/apel/summariser.log
```

Comment:
The logging section always comes last in our config files, so this should be moved below the new settings.